### ISSUE TITLE
Hook up "No Transactions" view

### ISSUE THREAD

**Issue by allendav** [Automattic/woocommerce-payments#97]
When there are no transactions available to the display, present something meaningful instead of an empty table.

cc @LevinMedia for a mock

Child of #82

**Comment by naman03malhotra** [comment]
Hi folks, I was thinking of picking up this issue as my 2nd "good first issue".

Looks like adding a prop to the `TableCard` component would give us the desired behaviour of passing a custom message as @reykjalin suggested. Can we discuss if this issue is still relevant before we start working on it? As it has been a while since the last activity.

**Comment by naman03malhotra** [comment]
Presently the empty table renderers with the default message `No data to display`. The `TableCard` component currently does not support the passing of a custom message.

Screenshot with default message on route `/payments/transactions`
![image](https://user-images.githubusercontent.com/15019298/115363089-c7fda580-a1df-11eb-832e-2f2e9862a347.png)

Screenshot with custom message on route `/payments/transactions`
![image](https://user-images.githubusercontent.com/15019298/115386112-9fcd7100-a1f6-11eb-8f7e-eaf7804ae571.png)

We can discuss if there is a requirement to pass a custom message as of now, then we have to make changes to the `TableCard` component in WCAdmin. 

IMO it would be a "good to have", not sure about the requirement of this feature.

Or if the present default message works for us, we can close this issue and re-open if the need arises.

cc @LevinMedia

**Comment by LevinMedia** [comment]
I think the current present default message works - thanks @naman03malhotra !

**Comment by jrodger** [comment]
I'm not sure what's going on with the "undefined transactions" displayed in the footer. I've seen that from time to time, but I thought it was limited to the loading state.

Would you mind creating a new issue so we can look into that @naman03malhotra?

What would you expect to be displayed there @LevinMedia? "0 transactions $0 total $0 fees $0 net", something like that? Or omitted entirely?

**Comment by naman03malhotra** [comment]
@jrodger I observed this issue as well, though it occurs intermittently. Initially, I thought there was a problem with my local setup, but doesn't look like it. I'll create an issue for the same and look into it. Thank you for confirming this!

For now, I am closing this issue as per the comment. https://github.com/Automattic/woocommerce-payments/issues/97#issuecomment-823369912
